python/lsst/ts/m2com/mock/mock_model.py

```diff
@@ -139,10 +141,16 @@ def __init__(
 
         self.telemetry_interval = telemetry_interval
 
-        self.inclination_source: MTM2.InclinationTelemetrySource = (
-            MTM2.InclinationTelemetrySource.ONBOARD
+        self.inclinometer_angle_external = (
+            self.control_open_loop.correct_inclinometer_angle(inclinometer_angle)
         )
 
+        self.control_parameters = {
```

Comment:
Isn't mypy complaining about this? you may want to add a type annotation. This might be defaulting to `dict[Any, Any]`, which is very broad. Maybe try `dict[str, bool | float]` ?

Reply:
There is no `mypy` complaints. Thanks!